lib/elixir/test/elixir/gen_server_test.exs

```diff
@@ -30,34 +30,31 @@ defmodule GenServerTest do
       # There is a race condition if the agent is
       # restarted too fast and it is registered.
       try do
-        self() |> Process.info(:registered_name) |> elem(1) |> Process.unregister
+        self() |> Process.info(:registered_name) |> elem(1) |> Process.unregister()
       rescue
         _ -> :ok
       end
+
       :ok
     end
   end
 
   test "generates child_spec/1" do
     assert Stack.child_spec([:hello]) == %{
-      id: Stack,
-      start: {Stack, :start_link, [[:hello]]}
-    }
+             id: Stack,
+             start: {Stack, :start_link, [[:hello]]}
+           }
```

Comment:
do these need to be extracted? or we reserving that for expressions within tuples (ie Example 1 from https://github.com/elixir-lang/elixir/issues/6643)?

Reply:
We don't need to extract this one too. We usually extract only when we don't want the code to take multiple lines. But in this case the extraction itself uses multiple lines (plus multiple lines are common in maps) so I wouldn't extract it.

Reply:
gotcha, will keep that in mind :)